Misc/NEWS.d/next/Library/2020-07-30-14-56-58.bpo-41440.rju34k.rst

```diff
@@ -0,0 +1 @@
+Add os.cpu_count() support for VxWorks RTOS.
```

Comment:
```suggestion
Add :func:`os.cpu_count()` support for VxWorks RTOS.
```

Reply:
Please update https://github.com/python/cpython/blob/master/Doc/whatsnew/3.10.rst also.

Reply:
For What's New, which section is suitable to add my changes? "Improved Modules" or "Porting to Python 3.10"?

Reply:
Porting to Python 3.10 should be more suitable so added there.